org.learningequality.Kolibri.appdata.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.learningequality.Kolibri</id>
+  <name>Kolibri</name>
+  <developer_name>Learning Equality</developer_name>
+  <summary>Offline education technology platform</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <url type="homepage">https://learningequality.org/kolibri/</url>
+  <url type="donation">https://learningequality.org/donate/</url>
+  <url type="help">https://kolibri.readthedocs.io/en/latest/</url>
+  <url type="bugtracker">https://github.com/learningequality/kolibri/issues</url>
+  <url type="translate">https://learningequality.org/translate/</url>
+  <description>
+    <p>Kolibri makes high quality education technology available in low-resource communities such as rural schools, refugee camps, orphanages, non-formal school systems, and prison systems. Running on a wide variety of hardware, it enables offline access to a rich set of content and tools to support learners, coaches, and administrators.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Browsing content channels in Kolibri</caption>
+      <image type="source" width="1544" height="806">https://learningequality.org/static/img/kickstarter/Kolibri-channels.png</image>
+    </screenshot>
+  </screenshots>
+  <categories>
+    <category>Education</category>
+    <category>Learning</category>
+    <category>Multimedia</category>
+    <category>Videos</category>
+    <category>Quiz</category>
+  </categories>
```

Comment:
Should be set on the destkop file. The metainfo shipped by Flathub uses appstream-compose to take that info from the desktop file

Reply:
Ok, yep! Moving there.

Reply:
Looks like these aren't valid desktop entry categories (except Education), so I'm removing them from there again (but leaving them off here too).